api/src/main/java/org/pmiops/workbench/api/WorkspacesController.java

```diff
@@ -601,15 +615,9 @@ public ResponseEntity<CloneWorkspaceResponse> cloneWorkspace(
 
     DbUser user = userProvider.get();
```

Comment:
createWorkspace above has 
`workspace.setCreator(user.getUsername());` do we need same code here?